packages/metrics-dotnet/MutationMetrics/Models/JsonReport.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Mutation.Metrics.Core.Models
+{
+    public struct JsonReport
+    {
+        public string SchemaVersion { get; set; }
+        public IDictionary<string, int> Thresholds { get; set; } 
+        public IDictionary<string, SourceFile> Files { get; set; }
```

Comment:
I would like to see TestFiles added as well:

https://github.com/stryker-mutator/mutation-testing-elements/blob/master/packages/report-schema/src/mutation-testing-report-schema.json#L113